src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheckTest.java

```diff
@@ -162,15 +161,85 @@ public void testArrayInitialization()
             throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(WhitespaceAroundCheck.class);
         checkConfig.addAttribute("tokens", "ARRAY_INIT");
+        checkConfig.addAttribute("tokens", "RCURLY");
```

Comment:
Please create a new test case for this

Reply:
Doubt- Why do I need to add `checkConfig.addAttribute("tokens", "RCURLY");` to make the test working? `RCURLY` is already there in the list of defaults and `checkConfig` is a reference variable to`DefaultConfiguration` .